CHANGELOG.md

```diff
@@ -62,6 +62,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Deprecate `.value` from container and object db listings
+  - `dexie` or `rxjs` removed this from db returns, couldn't find a change causing
+    this in either project, but this is the observed behavior
```

Comment:
It should be a change in `vueuse/rxjs`, and I looked through their changes, and didn't see it either '^^.

Reply:
Yep, I searched for any changes in related dependencies for quite long and couldn't see anything hinting towards it. The `.value` just vanished from the returned observers. The best guess from me at this point is that it might have something to do with `composition-api` being part of Vue by default in `2.7`, or some other backported feature.

Reply:
I haven't run into any issues with this approach yet, but it troubles me to not actually know the precise reason things have changed